lib/class-wp-rest-block-patterns-controller.php

```diff
@@ -0,0 +1,121 @@
+<?php
+/**
+ * REST API: WP_REST_Block_Patterns_Controller class
+ *
+ * @subpackage REST_API
+ * @package    WordPress
+ */
+
+/**
+ * Core class used to access block patterns via the REST API.
+ *
+ * @see   WP_REST_Controller
+ */
+class WP_REST_Block_Patterns_Controller extends WP_REST_Controller {
```

Comment:
All methods in this function need since documentation.

Reply:
All methods of the controller are documented, and I moved the controller to `compat/wordpress-6.0`. The endpoint should be moved to Core for 6.0 release.